src/app/progress/progress.module.ts

```diff
@@ -0,0 +1,22 @@
+import { module } from 'angular';
+import { IStateProvider } from 'angular-ui-router'
+import { react2angular } from 'react2angular';
+
+import bungieApiModule from '../bungie-api/bungie-api.module';
+import { ProgressService } from './progress.service';
+import { Progress } from './progress';
+
+// This is the Destiny 2 "Progress" page with milestones and factions.
+export default module('progressModule', [bungieApiModule])
+  .factory('ProgressService', ProgressService)
+  .component('d2Progress', react2angular(Progress, ['account'], ['ProgressService', '$scope', 'dimSettingsService']))
```

Comment:
Where is the account binding being set?

Reply:
This happens the same way it happens for our normal Angular components - UI-Router populates the binding based on a parent route (destiny-account) having a "resolve" for "account". You can see that our other components like "inventory" get the account magically in their bindings as well.